eng/testing/WasmRunnerTemplate.sh

```diff
@@ -10,7 +10,7 @@ XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 dotnet xharness wasm test --engine=$JAVASCRIPT_ENGINE \
     --js-file=runtime.js -v \
     --output-directory=$XHARNESS_OUT \
-    -- --enable-gc --run WasmTestRunner.dll ${@:2}
```

Comment:
this moved to tests.mobile.targets in master, you'll need to make the change there to resolve the merge conflict.